plugins/org.eclipse.n4js.cli/src/org/eclipse/n4js/cli/lsp/LspServer.java

```diff
@@ -208,4 +232,46 @@ public void close() throws IOException {
 		Logger.getRootLogger().addAppender(appender);
 		return appender;
 	}
+
+	private Appender copyLog4jErrorsToDiagnosisLogger(DiagnosisLogger diagnosisLogger) {
+		Appender appender = new DiagnosisLoggerAppender(diagnosisLogger);
+		Logger.getRootLogger().addAppender(appender);
+		return appender;
+	}
+
+	private static final class DiagnosisLoggerAppender extends AppenderSkeleton {
```

Comment:
You could say in a comment which classes etc. were planned to be a temporary solution